<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="swapnil-PC" failures="1" tests="6" name="com.phpTravels.qa.testcases.HomePageTest" time="113.566" errors="0" timestamp="29 Jan 2020 18:47:55 GMT" skipped="5">
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="quickHomePageHotelLinkPageTest" time="0.000">
    <skipped/>
  </testcase> <!-- quickHomePageHotelLinkPageTest -->
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="setUp" time="113.566">
    <failure message="expected [PHPTRAVELS | Travel Technology Partner] but found [www.phptravels.net | 524: A timeout occurred]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [PHPTRAVELS | Travel Technology Partner] but found [www.phptravels.net | 524: A timeout occurred]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.phpTravels.qa.base.TestBase.initialization(TestBase.java:75)
	at com.phpTravels.qa.testcases.HomePageTest.setUp(HomePageTest.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- setUp -->
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="HotelLinkPageTest" time="0.000">
    <skipped/>
  </testcase> <!-- HotelLinkPageTest -->
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="selectCheckinDateTest" time="0.000">
    <skipped/>
  </testcase> <!-- selectCheckinDateTest -->
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="searchHoteslsTest" time="0.000">
    <skipped/>
  </testcase> <!-- searchHoteslsTest -->
  <testcase classname="com.phpTravels.qa.testcases.HomePageTest" name="namePriceHoteslsTest" time="0.000">
    <skipped/>
  </testcase> <!-- namePriceHoteslsTest -->
</testsuite> <!-- com.phpTravels.qa.testcases.HomePageTest -->
